notifications.bs

```diff
@@ -696,7 +699,8 @@ constructor steps are:
  <a>throw</a> a {{TypeError}}.
 
  <li><p>Let <var>notification</var> be the result of <a>creating a notification</a> given
- <var>title</var> and <var>options</var>. Rethrow any exceptions.
+ <var>title</var>, <var>options</var>, and <a>this</a>'s <a>relevant settings object</a>.
```

Comment:
They're equivalent in constructors, but using this's relevant settings object here and current settings object a few lines up is a bit confusing.